NOTES-ANDROID.md

```diff
@@ -59,16 +59,16 @@ Notes for Android platforms
  conflict, and mixing the two is therefore not supported. Migration to
  `CROSS_SYSROOT`-less setup is recommended.
 
- One can engage clang by adjusting PATH to cover same NDK's clang. Just
+ One can engage clang by adjusting PATH to cover the same NDK's clang. Just
  keep in mind that if you miss it, Configure will try to use gcc...
  Also, PATH would need even further adjustment to cover unprefixed, yet
  target-specific, ar and ranlib. It's possible that you don't need to
  bother, if binutils-multiarch is installed on your Linux system.
 
  Another option is to create so called "standalone toolchain" tailored
  for single specific platform including Android API level, and assign its
- location to `ANDROID_NDK_ROOT`. In such case you have to pass matching
- target name to Configure and shouldn't use `-D__ANDROID_API__=N`. `PATH`
+ location to `ANDROID_NDK_ROOT`. In such case, you have to pass matching
+ target name to configure and shouldn't use `-D__ANDROID_API__=N`. `PATH`
```

Comment:
The script's name is `Configure` with a capital.  This change is therefore incorrect.